src/modules/discovery/ports.py

```diff
@@ -20,9 +20,10 @@ def __init__(self, event):
         self.port = event.port
 
     def execute(self):
-        logging.debug("host {0} try ports {1}".format(self.host, default_ports))
+        logging.debug("host {0} try reach ports: {1}".format(self.host, default_ports))
```

Comment:
Not quite English - either "try ports" or "try to reach ports" (I think I prefer "try ports" as it's more succinct)